miio/integrations/vacuum/roborock/vacuumcontainers.py

```diff
@@ -953,3 +979,40 @@ def current_high(self) -> int:
     @property
     def current_integral(self) -> int:
         return self.data["current_integral"]
+
+
+class MopDryerSettings(DeviceStatus):
+    """Container for mop dryer add-on."""
+
+    def __init__(self, data: Dict[str, Any]):
+        # {'status': 0, 'on': {'cliff_on': 1, 'cliff_off': 1, 'count': 10, 'dry_time': 10800},
+        # 'off': {'cliff_on': 2, 'cliff_off': 1, 'count': 10}}
+        self.data = data
+
+    @property
+    @setting(
+        "Mop dryer enabled",
+        setter_name="set_mop_dryer_enabled",
+        icon="mdi:tumble-dryer",
+        entity_category="config",
+        enabled_default=False,
+    )
+    def enabled(self) -> bool:
+        """Return if mop dryer is enabled."""
+        return self.data["status"] == 1
+
+    @property
+    @setting(
+        "Mop dry time",
+        setter_name="set_mop_dryer_dry_time",
+        icon="mdi:fan",
+        unit="hour",
+        min_value=2,
+        max_value=4,
+        step=1,
+        entity_category="config",
+        enabled_default=False,
+    )
+    def dry_time(self) -> bool:
+        """Return mop dry time."""
```

Comment:
What is the meaning of dry_time? The setting for how long it will be drierd?

The return value should likely be timedelta here, too.

Reply:
Also changed to timedelta.